packages/seal-meilisearch-adapter/src/MeilisearchIndexer.php

```diff
@@ -29,6 +29,7 @@ public function __construct(
         private readonly Client $client,
     ) {
         $this->marshaller = new Marshaller(
+            dateFormat: 'U',
```

Comment:
if I understood correctly https://www.meilisearch.com/docs/learn/filtering_and_sorting/working_with_dates

sorting works by date strings, but for filters we need save a unixtimestamp. So I use like in Loupe and Typesense a UnixTimestap now.